proxy/cmd/main.go

```diff
@@ -18,6 +18,7 @@ func main() {
 	var targetVncPort = flag.String("targPort", "", "target vnc server port (deprecated, use -target)")
 	var targetVncHost = flag.String("targHost", "", "target vnc server host (deprecated, use -target)")
 	var targetVncPass = flag.String("targPass", "", "target vnc password")
+	var dynamicLookup = flag.Bool("dynamic-lookup", false, "lookup target UNIX socket path based on WebSocket URI")
```

Comment:
it feels strange to have a different kind of flag using kebap-case instead of pascalCase.

Reply:
Fixed.